multimodal/src/autogluon/multimodal/utils/onnx.py

```diff
@@ -48,7 +48,7 @@ class OnnxModule(object):
     so that we can predict with TensorRT by simply replacing predictor._model with OnnxModule.
     """
 
-    def __init__(self, onnx_path: str, providers: Optional[Union[dict, List[str]]] = None):
+    def __init__(self, onnx_path: Union[str, bytes], providers: Optional[Union[dict, List[str]]] = None):
```

Comment:
Update the docstring, and describe what bytes it is expecting

Reply:
docstring updated.